src/arduino/exampleProvider.ts

```diff
@@ -43,7 +43,7 @@ export class ExampleProvider implements vscode.TreeDataProvider<ExampleItem> {
         this._exmaples = null;
         this._exampleManager.loadExamples().then((examples) => {
             this._exmaples = examples;
-            this._onDidChangeTreeData.fire();
+            this._onDidChangeTreeData.fire(null);

```

Comment:
I'm guessing this is unrelated to anything CG related?

Reply:
This line broke when I updated the dependencies, so I patched it to continue working.

I'm not sure this code is even used, as it's called "Example Provider" - My assumption is that it's part of the original vscode extension template or something.